### PR TITLE
Fix bad substitution error when plan contains sensitive characters

### PR DESCRIPTION
**Pull request by ryanwholey**
got an [ambiguous error](https://github.com/TakeScoop/aws-golden-gate/runs/3499985140?check_suite_focus=true) from this action, not much to go on.

```
/home/runner/work/_temp/40f20cf6-23bf-45f5-9983-2c3a24ec8e9d.sh: command substitution: line 73: syntax error near unexpected token `('
Error: Process completed with exit code 2.
```

Was able to find the action script that the action was using and noticed that the `${{ inputs.plan }}` portion is templated into the script, which makes sense. After running the script locally, I can see that this has to do with bash trying to interpret part of the plan:

```
./make: line 13: syntax error near unexpected token `('
./make: line 13: `      - description  = "The environment the workspace provisions (automatically set to the name suffix)" -> null'
```

As a workaround, we can store the value of the plan in a variable and use that for jq intead of the string template representation. (fixed [example](https://github.com/TakeScoop/aws-golden-gate/actions/runs/1195709102))

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -24,6 +29,7 @@ runs:
           --monochrome-output \
           --arg plan "${{ inputs.plan }}" \
```
Comment (bendrucker) [inline action.yml:25]: Using single quotes here should do it, no? Singles are for string literals, doubles can contain substitution sequences.

Comment (ryanwholey) [reply action.yml:25]: oh yeah if its a templating already than that might work.. lets see

Comment (ryanwholey) [reply action.yml:25]: yep that works
https://github.com/TakeScoop/aws-golden-gate/runs/3500486843?check_suite_focus=true

**Review by bendrucker — APPROVED** [review]
Nice, UUOC is a shell meme:

https://porkmail.org/era/unix/award.html

Sometimes cat is worth it for readability but often it's a smell worth removing

**Pull request merged**